packages/multi-test/src/app.rs

```diff
@@ -903,4 +911,172 @@ mod test {
             .unwrap();
         assert_eq!(state.beneficiary, random);
     }
+
+    mod replay_data_overwrite {
+        use cosmwasm_std::to_binary;
+
+        use crate::test_helpers::EmptyMsg;
+
+        use super::*;
+
+        fn make_echo_submsg(
+            contract: Addr,
+            data: impl Into<Option<&'static str>>,
```

Comment:
Woah, so `"name"` will implement `Into<Option<&str>>` ?

Meaning, I can use `"data"` or `None` rather than `Some("data")` / `None`?

Cool trick

Reply:
Yes, and if you are really hard working you can still use `Some("data")`.